release/ray_release/scripts/ray_bisect.py

```diff
@@ -43,20 +44,28 @@
         "flakiness. Commit passes only when it passes on all runs"
     ),
 )
+@click.option(
+    "--is-full-test",
+    is_flag=True,
+    show_default=True,
+    default=False,
```

Comment:
shouldn't this be true? why the smoke version is default?

also, is it true that all tests have a smoke version? or just some tests?

Reply:
i intentionally make the default smoke test, only a few tests have smoke version; those that have smoke version typically run for extended longer hour in its full version (like for a day) so bisecting those by default will burn a lot of money